qa/workunits/rbd/cli_generic.sh

```diff
@@ -743,6 +743,14 @@ test_migration() {
     rbd migration execute test1
     rbd migration commit test1
 
+    # testing trash
```

Comment:
Nit: avoid these tests when using image format V1

Reply:
I think the image format of 'test1' is V2 at here (since `rbd migration prepare test1 --data-pool rbd2` above), am i right?

Reply:
Ah, yes -- the data pool kicks it to format v2 implicitly.